inference-engine/include/cpp/ie_memory_state.hpp

```diff
@@ -58,12 +58,17 @@ class MemoryState {
      * Wraps IMemoryState::GetLastState
```

Comment:
we also need to add second optional parameter to `explicit MemoryState(IMemoryState::Ptr pState)`:
```c++
explicit MemoryState(IMemoryState::Ptr pState, details::SharedObjectLoader::Ptr plg = {})
```
And need to add a test for such case (see `inference-engine/tests/functional/plugin/shared/src/behavior/cpp_holders.cpp` as a reference)

Reply:
Ctor was updated, test extending is in progress